src/scenes/Event/store.js

```diff
@@ -295,15 +306,28 @@ export default class EventStore {
 
   @action
   queryTransactions = async (address) => {
-    this.transactions = await queryAllTransactions(
-      [{ topicAddress: address }],
+    const pendings = await queryAllTransactions(
+      [{ topicAddress: address, status: TransactionStatus.PENDING }],
       { field: 'createdTime', direction: SortBy.DESCENDING },
     );
+    this.withdraws = await queryWithdraws([{ topicAddress: address }]);
```

Comment:
oh thought you were going to remove the individual arrays for each type? are the individual arrays used somewhere? if not, I wouldn't bother storing them.

Reply:
I will to remove them eventually.